### PR TITLE
Rename AnnounceCardFilter to AnnounceCard

### PR DESCRIPTION
**Pull request by edo9300**
https://github.com/Fluorohydride/ygopro-core/pull/216

### CODE REVIEW

File: c50078320.lua [hunk 1]
```diff
@@ -37,15 +37,15 @@ function c50078320.operation(e,tp,eg,ep,ev,re,r,rp)
 	Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_CODE)
 	if bit.band(cv,ANNOUNCE_CARD)~=0 then
 		--c:IsType(cv) and not c:IsCode(code)
-		ac=Duel.AnnounceCardFilter(tp,cv,OPCODE_ISTYPE,code,OPCODE_ISCODE,OPCODE_NOT,OPCODE_AND)
+		ac=Duel.AnnounceCard(tp,cv,OPCODE_ISTYPE,code,OPCODE_ISCODE,OPCODE_NOT,OPCODE_AND)
```
Comment (mercury233) [inline c50078320.lua]: this card need more changes

Comment (edo9300) [reply c50078320.lua]: Why? _if changes are needed that is because it was wrong in first place then_

Comment (mercury233) [reply c50078320.lua]: `ANNOUNCE_CARD` or `ANNOUNCE_CARD_FILTER` should be obsolete

Comment (edo9300) [reply c50078320.lua]: In fact I removed AnnounceCardFilter, and merged the 2 functions

**Pull request merged**